config/jobs/image-pushing/k8s-staging-test-infra.yaml

```diff
@@ -312,7 +312,46 @@ postsubmits:
           - --scratch-bucket=gs://k8s-staging-test-infra-gcb
           - --project=k8s-staging-test-infra
           - images/git-custom-k8s-auth/
-
+    - name: post-test-infra-push-misc-images-canary
+      cluster: k8s-infra-prow-build-trusted
+      run_if_changed: '^(\.ko\.yaml|hack/(make-rules|prowimagebuilder)|label_sync/.+\.go|robots/commenter|robots/pr-creator|robots/issue-creator|testgrid/cmd|gcsweb)'
+      decorate: true
+      labels:
+        # Building deck requires docker for typescript compilation.
+        preset-dind-enabled: "true"
+      branches:
+      - ^master$
+      max_concurrency: 1
+      spec:
+        serviceAccountName: gcb-builder
+        containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240705-131cd74733-test-infra
```

Comment:
we should run this image/ command in a cloudbuild.yml (configured for the 32 core VM type) and put the prowjob as another standard image pushing job invoking it with the image-builder tool (See docs at https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io#managing-kubernetes-container-registries => https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io#enabling-automatic-builds)

Reply:
Thanks! Updated with a cloudbuild.yaml and a more standard job. (I did not test the job itself, but I did verify the cloudbuild.yaml works when run with gcloud (against the k8s-testgrid project).